api/src/controllers/download/datasets-controller.ts

```diff
@@ -56,10 +56,7 @@ export class DatasetsController extends BaseController {
         // for data rendering logic
         "file",
         "integration",
-        {
-          association: "fields",
-          where: { isExcludedFromSearch: false },
-        },
```

Comment:
Maybe I shouldn't have remove this? When I add it back I should add a test explaining why its written this way. Maybe it should have been "required: false", or maybe this should also be a policy scope check?

Reply:
As this is handled in the "download", and download is only available after you gain full access, it is safe to return all fields here.